src/main/java/org/example/examplefinalProject/controller/TeacherController.java

```diff
@@ -29,4 +32,24 @@ public ModelAndView detail(@PathVariable int teacherId) {
         modelAndView.addObject("teacher", teacherService.findById(teacherId));
         return modelAndView;
     }
+
+    @PostMapping("/createTeacher")
+    public String createTeacher(@RequestParam String teacherName,
+                                @RequestParam String teacherSurname,
+                                @RequestParam String classRoomName) {
+        teacherService.createTeacher(teacherName, teacherSurname, classRoomName);
+        //log.info(String.format("User created teacher '%s' '%s'", teacherName, teacherSurname));
```

Comment:
do we need this?

Reply:
Not anymore.
It was used for the development.